tensornetwork/network.py

```diff
@@ -939,13 +953,18 @@ def split_node_full_svd(self,
       max_truncation_err: The maximum allowed truncation error.
 
     Returns:
-      left_node: The new left node created. Its underlying tensor is the same
+    A tuple containing:
+      left_node: 
+        The new left node created. Its underlying tensor is the same
         as the U tensor from the SVD.
-      singular_values_node: The new node representing the diagonal matrix of
+      singular_values_node: 
+        The new node representing the diagonal matrix of
         singular values.
-      right_node: The new right node created. Its underlying tensor is the same
+      right_node: 
+        The new right node created. Its underlying tensor is the same
         as the V* tensor from the SVD.
-      truncated_singular_values: The vector of truncated singular values.
+      truncated_singular_values: 
+        The vector of truncated singular values.
```

Comment:
The language in the descriptions of the nodes and `truncated_singular_values` should be very similar between `split_node` and `split_node_full_svd`. I'm partial to something like "A new node created that connects to all of the `left_edges`. Its underlying tensor is the same as the U tensor from the SVD." and so on, with appropriate changes for the `split_node` method that mentions sqrt(S).

Reply:
Added.